scrapers/ModelMediaUS.yml

```
@@ -0,0 +1,27 @@
+name: "ModelMediaUS"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - modelmediaus.com/videos/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h3/text()
+      Date:
+        selector: //td[contains(.,"Released:")]/following-sibling::td/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
```

```suggestion
          - parseDate: Jan 2, 2006
```
Single digit day dates  dont seem to have 0 padding